File: 0001/solution.cpp
#include <iostream>

int solution() {
  int sum = 0;
  for (int i = 0; i < 1000; i++)
    if (i % 3 == 0 || i % 5 == 0) sum += i;
  return sum;
}

int main(int argc, char ** argv) {
  std::cout << solution() << std::endl;
  return 0;
}

File: 0001/solution.py
#!/usr/bin/python

"""Find the sum of all multiples of 3 OR 5 under 1000.
"""
def solution():
    values = [x for x in range(0,1000) if  x % 3 == 0 or x % 5 == 0]
    return sum(values)


if __name__ == "__main__":
    print(solution())
